src/@next/components/organisms/CreditCardGrid/CreditCardGrid.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import { AddNewTile, TileGrid } from "@components/atoms";
+import { CreditCardTile } from "@components/molecules";
+
+import { IProps } from "./types";
+
+export const CreditCardGrid: React.FC<IProps> = ({ creditCards }: IProps) => {
+  const addNewTile = [<AddNewTile type="card" />];
+  const ccTiles = creditCards.map(cc => {
```

Comment:
I would simplify the code and get rid of the `return` keyword from the next line.
```suggestion
  const ccTiles = creditCards.map(cc => (
```